client/my-sites/email/paths.js

```diff
@@ -298,3 +298,31 @@ export function emailManagementInbox( siteName = null ) {
 export function isUnderEmailManagementAll( path ) {
 	return path?.startsWith( emailManagementAllSitesPrefix + '/' );
 }
+
+/**
+ *
+ * @param { string } ownerUserName The username that will appear in the log in screen
+ * @param { string | null | undefined } selectedSiteSlug The selected site slug
+ * @param { string } selectedDomainName The domain name to purchase email subscription
+ * @returns {string} Login Url with email_address and redirect_to parameters included in the URI
+ */
+export function loginUrlWithUserNameAndRedirectToEmailProvidersComparison(
```

Comment:
Maybe we shouldn't add this as a utility function in this file. Letero doesn't own the `/log-in` slug, and to my mind, this file is a single source of truth for slugs related to email management in Calypso.

Reply:
That's a good point, I'll include it in the new component developed.

Reply:
Looks like you maybe forgot to remove this? 🙂